### PR TITLE
Bevy's 4th birthday: Reflecting on a dream job

### PR DESCRIPTION
**Pull request by alice-i-cecile**
This post is a reflection on the year behind, the year ahead, and my first few months working at the Bevy Foundation.

I'd like to publish this at the start of September, probably about a week before the roundup of posts.

Critical feedback and editing welcome: this is a more personal tone, but I know it needs tightening up.

# To do

- [x] fix title of file
- [x] add summary

### CODE REVIEW

**Review by cart — APPROVED** [review]

File: content/news/2024-??-??-dream-job/index.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
++++
+title = "I landed my dream job making a Rust game engine. Now what?"
+date = 2024-09-TODO
```
Comment (alice-i-cecile) [inline content/news/2024-??-??-dream-job/index.md:3]:
```suggestion
date = 2024-09-10
```

**Comment by alice-i-cecile** [comment]
Awesome, thanks for reviewing! I'll merge and publish this tomorrow :)

**Review by Jondolf — COMMENTED** [review]
Huh, looks like I reviewed this a month ago, but never submitted it. Contents look good, just some typos and wording things :)

(feel free to ignore)

File: content/news/2024-??-??-dream-job/index.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+I would focus on that when I was feeling well, get done what I could, and as soon as my focus broke or the task was completed, that was it. I was "done work" for the day.
+
+To complement this, the strict 9-5 M-F schedule would be relaxed. I could work whenever I felt like it (turns out, for me, that's quite a bit):
+tackling the million little things that help keep Bevy ticking along.
+But there would be no *obligation*: no sense that I *must* do these things, or must do them in a timely fashion.
+I could be there for the impromptu late night design chats, but still disconnect during date night because, after all, I finished work *hours* ago.
+
+This balance has worked really well for me: letting me drive forward larger initiatives (`bevy_color`! A huge `leafwing-input-manager` refactor! The 0.14 release!) without falling behind on the flood of notifications.
+Would I recommend you use it? Probably not! It requires a egree of flexibility that many organizations won't afford you, you need to be driven and self-motivated and frankly, the work-life boundary is far blurrier than most people can live with.
```
Comment (Jondolf) [inline content/news/2024-??-??-dream-job/index.md:56]:
```suggestion
Would I recommend you use it? Probably not! It requires a degree of flexibility that many organizations won't afford you, you need to be driven and self-motivated and frankly, the work-life boundary is far blurrier than most people can live with.
```

**Pull request merged**